ingenialink/ethercat/network.py

```diff
@@ -174,7 +174,7 @@ class EthercatNetwork(Network):
 
     MANUAL_STATE_CHANGE = 1
 
-    DEFAULT_ECAT_CONNECTION_TIMEOUT_S = 1
+    DEFAULT_ECAT_CONNECTION_TIMEOUT_S = 2
```

Comment:
I suppose this corresponds to this field of the esi file, it's good that we are aligned in what we're generating

<img width="546" height="339" alt="Image" src="https://github.com/user-attachments/assets/ea32ba93-94a4-440e-9bef-d03a7a75d003" />